### PR TITLE
Feature/integrate travis ci

### PR DESCRIPTION
**Pull request by MJLHThomassen**
Linux builds pass. Windows builds still have missing header file issues. Investigation into this is ongoing.

### CODE REVIEW

**Comment by MJLHThomassen** [comment]
For some reason this broke someting in the win64 build in the code referencing libmad, still investigating...

File: README.md [hunk 1]
```diff
@@ -48,7 +48,7 @@ and then copying the following files from your VM onto your computer:
 * Git Pull this repository onto a Ubuntu flavoured Linux machine or VM
 * Terminal to the repository directory and run: ./install_deps.sh. This will install all development dependencies for running make
 * Run make to build for Linux 64 bits, find the output files in the build directory
-  * Run ./make-i368.sh for building the Linux 32 bits variant
+  * Run ./make-i386.sh for building the Linux 32 bits variant
```
Comment (solarisstar) [inline README.md:52]: thanks

File: README.md [hunk 2]
```diff
@@ -62,7 +62,7 @@ and then copying the following files from your VM onto your computer:
     * build/release-PLATFORM-ARCH/rpgxEF/qagameARCH
     * build/release-PLATFORM-ARCH/rpgxEF/cgameARCH
 
-* Be sure to run the game with the following cvars: +set vm_game 0 +set vm_ui 0 +set vm_cgame 0 +set fs_game RPG-X2
+* ~~Be sure to run the game with the following cvars: +set vm_game 0 +set vm_ui 0 +set vm_cgame 0 +set fs_game RPG-X2~~ UPDATE: cvars should no longer be required.
```
Comment (solarisstar) [inline README.md]: Just delete the line then 😄

File: cross-make-mingw.sh [hunk 3]
```diff
@@ -1,24 +1,24 @@
...
-    done
-fi
+done
 
-if [ "X$WINDRES" = "X" ]; then
-    for check in $CMD_PREFIX; do
-        full_check="${check}-windres"
+# Define WINDRES to the correct (32 or 64 bits) version of mingw
+for check in $CMD_PREFIX; do
+	full_check="${check}-windres"
```
Comment (solarisstar) [inline cross-make-mingw.sh]: Can this be combined into the first loop?

File: cross-make-mingw64.sh [hunk 4]
```diff
@@ -1,24 +1,24 @@
...
 	fi
-    done
-fi
+done
 
-if [ "X$WINDRES" = "X" ]; then
-    for check in $CMD_PREFIX; do
-        full_check="${check}-windres"
+# Define WINDRES to the correct (32 or 64 bits) version of mingw
+for check in $CMD_PREFIX; do
```
Comment (solarisstar) [inline cross-make-mingw64.sh]: Can this be combined into the first loop?

File: README.md (hunk 2 above)
Comment (MJLHThomassen) [reply README.md]: Fixed

File: cross-make-mingw.sh (hunk 3 above)
Comment (MJLHThomassen) [reply cross-make-mingw.sh]: Fixed

File: cross-make-mingw64.sh (hunk 4 above)
Comment (MJLHThomassen) [reply cross-make-mingw64.sh]: Fixed

**Pull request closed without merging**